app/pages/events/event_list/event_list.php

```diff
@@ -0,0 +1,79 @@
+<?php
+include_once __DIR__ . "/RenderEvents.php";
+
+restrict_access();
+$can_edit = check_auth(Access::$ADD_EVENTS);
+
+formatter("d MMM");
+$user = User::getCurrent();
+$events = Event::listAllOpen($user->id);
```

Comment:
Remove this, as it is not being used anymore.

Reply:
✅